distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalogValidation.java

```diff
@@ -94,115 +103,108 @@ public void beforeExam() throws Exception {
         }
     }
 
+    @Before
+    public void setup() throws Exception {
+        configureShowInvalidMetacards("true", "true", getAdminConfig());
+        configureFilterInvalidMetacards("false", "false", getAdminConfig());
+        configureEnforceValidityErrorsAndWarnings("true", "false", getAdminConfig());
+        configureMetacardValidityFilterPlugin(Arrays.asList(""), getAdminConfig());
+        console.runCommand("catalog:removeall -f");
+        console.runCommand("catalog:removeall -f --cache");
+        //verify catalog is clean
+        expect("Failed to clear catalog").within(10, TimeUnit.SECONDS)
+                .checkEvery(1, TimeUnit.SECONDS)
+                .until(() -> {
+                    try {
+                        String query = new CswQueryBuilder().addAttributeFilter(PROPERTY_IS_LIKE,
+                                "AnyText",
+                                "*")
+                                .getQuery();
+                        ValidatableResponse response = given().header(HttpHeaders.CONTENT_TYPE,
+                                MediaType.APPLICATION_XML)
+                                .body(query)
+                                .post(CSW_PATH.getUrl())
+                                .then();
+                        response.body(hasXPath(
+                                "/GetRecordsResponse/SearchResults[@numberOfRecordsMatched=\"0\"]"));
+                        return true;
+                    } catch (Exception e) {
+                        return false;
+                    }
+
+                });
+    }
+
     @Test
     public void testEnforceValidityErrorsOnly() throws Exception {
-
         //Configure to enforce validator
         configureEnforcedMetacardValidators(Collections.singletonList("sample-validator"),
                 getAdminConfig());
 
         //Configure to enforce errors but not warnings
         configureEnforceValidityErrorsAndWarnings("true", "false", getAdminConfig());
```

Comment:
Yes! That is one of the primary causes of instability in this test class. We can't directly check to see when the config changes make it to their respective services/beans so we added the retry logic on ingest and tests to get around that uncertainty.